tracr/io/read.py

```diff
@@ -1,11 +1,41 @@
 # tries to guess the format of the file
+"""
+Read in TIF or DCM intensity data and send to appropriate reader depending on
+format. Assume that a folder input contains data frames directly below
+(i.e. there are no nested folders.)
+
+INPUT:
+    - TIF: file (single or multilayer), or folder of TIF frames
```

Comment:
Does this assume that the image is read from bottom-to-top, i.e. frame-0000.tif is the bottom to frame-1024.tif is the top?